sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/statsbeat/CustomDimensions.java

```diff
@@ -86,4 +76,13 @@ private static OperatingSystem initOperatingSystem() {
             return OperatingSystem.OS_UNKNOWN;
         }
     }
+
+    // package protected - visible for test
```

Comment:
I would prefer not to modify the method visibility for testing because it leads to test a part of the implementation, not the global expected behavior, and this can make the code difficult to refactor later.